examples/platform-demo/platform/large.tfvars

```diff
@@ -0,0 +1,35 @@
+lb_config = {
+  "default" = {
+    "load_balancer_type" = "application",
+    "create_alb"         = true
+  }
+}
+
+postres_config     = {}
+common_name_prefix = "prod"
+ecs_config = {
+  "default" : {
+    name : "vote",
+    engine : {
+      type : "FARGATE",
+      default_weight : 100,
+      spot_weight : 0,
+    }
+  }
+}
+environment = "testing"
```

Comment:
Just wondering. Should this really be in the profile? I guess it depends on how we define the profile (i.e. by performance - small/large or by kind dev/prod). But what if the user needs large-dev etc. It quickly becomes multi-dimensional.
Unless we support multiple profiles that complement each other (e.g. generate --profile def --profile large). The assumption is that the profiles would not overlap or they would end up overriding each other.

Reply:
@kanaksinghal @dshelley66

Reply:
you are right, it might make more sense to have dev/staging/prod profiles. for now (with one profile model), it's still possible to create a profile say large-dev. we can consider adding multi dimensional profiles in future as we encounter the use-case.